scripts/build-windows-installer.ps1

```
@@ -191,6 +191,33 @@ function Get-FileHashHex([string]$Path) {
     return (Get-FileHash -LiteralPath $Path -Algorithm SHA256).Hash.ToLowerInvariant()
 }
 
+function Set-WindowsExecutableResources(
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Set-WindowsExecutableResources' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Set-WindowsExecutableResources'

(PSUseSingularNouns)